src/eggd_generate_bed.sh

```
@@ -70,8 +70,10 @@ main() {
         dx-jobutil-report-error "Error: empty bed file generated"
         exit 1
     else
-        sort -k1,1V -k2,2n "$bed_file" -o"temp_bed.bed"
+        sort -k1,1V -k2,2n "$bed_file" -o "temp_bed.bed"
         mv "temp_bed.bed" "$bed_file"
+        bed_file_content=$(cat "${bed_file}")
+        printf "%s\n%s" "${header_info}" "${bed_file_content}" > "${bed_file}"
```

Shellcheck (0.11.0)

[warning] header_info is referenced but not assigned.

(SC2154)